src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+</Steps>
+
+## Change your continuous payment settings
+
+A continuous payment is a payment that recurs at your set rate for as long as you are active on a web monetized page. When you leave the page, the payments stop.
+
+Continuous payments are enabled by default and sent at a rate of $0.60 USD per hour. Use the instructions below to change the rate or disable continuous payments. 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
+    <br /><img src="" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Rate* tab.
+    <br /><img
+    src=""
+    alt="Rate tab with rate of pay field and continuous payment toggle"
+    style="max-width:300px"
+    class="img-outline"
+    />
+3. Enter the amount the extension will send, per hour, to web monetized sites.
+4. Select the **Continuous payment** toggle to enable/disable continuous payments. This is a global setting that applies to all web monetized pages you visit.
+</Steps>
+
+## Make a one-time payment 
+
+You can make a one-time payment (OTP) to a page instead of, or in addition to, continuous payments. The amount of the OTP is deducted from your budget. 
+
+<Steps>
+1. Ensure the URL shown above the amount field is the page you intend to pay. 
+    <br /><img
+    src=""
+    alt="Extension showing an available balance of five dollars, with one dollar in the amount field and send now button available"
+    style="max-width:300px"
+    class="img-outline"
+    />
+2. Enter the amount and click **Send now**. The button's text will briefly change to *Payment successful*, then back to *Send now*.
+    <br /><img
+    src=""
+    alt="Extension showing an available balance of four dollars, with the amount field set to zero and payment successful showing in button"
+    style="max-width:300px"
+    class="img-outline"
+    />
+</Steps>
+
+## Resolve a wallet connection error
+
+When you try to connect the extension to your digital wallet, the extension attempts to automatically connect with your wallet provider to add a key to your wallet account. 
+
+You'll receive an error if the connection fails. When this happens, you must copy the key that appears in the extension and manually add it to your wallet account. The steps for adding the key to your account depend on who your wallet provider is. 
+
+### Interledger Wallet
+
+<Steps>
+1. Sign in to your Interledger Wallet.
+2. Select **Settings** from the left menu.
+3. Select **Keys**.
+4. Click **Add a public key**.
+5. Enter a nickname for the key, then paste the key into the public key field.
+6. Click **Save**.
+7. Go back to your extension and try connecting again.
+</Steps>
+
+Contact Interledger Wallet at support@interledger.app if you need additional help.
+
+## Disconnect the extension from your wallet
+
+When you disconnect the extension from your wallet, the extension can no longer make payments. You can always reconnect your wallet later.
```

Comment:
Should we tell user that the left-over amount stays within your wallet and is not thrown away?